app/styles/component/_affiliate-unit.scss

```diff
@@ -0,0 +1,43 @@
+.affiliate-unit {
+  &__header {
+    font-weight: $wds-font-weight-medium;
+    text-transform: uppercase;
+  }
+
+  &__link {
+    background-size: cover;
+    display: block;
+    height: 222px;
+    position: relative;
+
+    &::after {
+      content: '';
+      display: block;
+      padding-top: (4 / 3) * 100%; /* 4:3 aspect ratio */
+    }
+  }
+
+  &__content {
+    background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), #000);
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: flex-end;
+    left: 0;
+    padding: 18px 14px;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  &__heading {
+    color: #fff;
+    font-weight: bold;
```

Comment:
Instead of using scss for typography mobile-wiki seems to be using class names - ie. `.wds-font-weight-bold` and `.wds-leading-tight`

https://fandomdesignsystem.com/base-styles/typography